src/Microsoft.AspNetCore.Mvc.Formatters.Json/JsonOutputFormatter.cs

```diff
@@ -25,16 +24,11 @@ public class JsonOutputFormatter : TextOutputFormatter
         // the serializer and invalidate it when the settings change.
         private JsonSerializer _serializer;
 
-        public JsonOutputFormatter()
-            : this(SerializerSettingsProvider.CreateSerializerSettings(), ArrayPool<char>.Shared)
-        {
-        }
-
-        public JsonOutputFormatter(JsonSerializerSettings serializerSettings)
-            : this(serializerSettings, ArrayPool<char>.Shared)
-        {
-        }
```

Comment:
Removing these wasn't mentioned in #4339 but seems to fit. Let me know if you disagree.